spotinst/ocean_aks_np/fields_spotinst_ocean_aks_np.go

```diff
@@ -254,6 +254,78 @@ func Setup(fieldsMap map[commons.FieldName]*commons.GenericField) {
 		},
 		nil,
 	)
+
+	fieldsMap[UpdatePolicy] = commons.NewGenericField(
+		commons.OceanAKSNP,
+		UpdatePolicy,
+		&schema.Schema{
+			Type:     schema.TypeList,
+			Optional: true,
+			MaxItems: 1,
+			Elem: &schema.Resource{
+				Schema: map[string]*schema.Schema{
+					string(ShouldRoll): {
+						Type:     schema.TypeBool,
+						Required: true,
+					},
+					string(ConditionedRoll): {
+						Type:     schema.TypeBool,
+						Optional: true,
+					},
+					string(ConditionedRollParams): {
+						Type:     schema.TypeList,
+						Elem:     &schema.Schema{Type: schema.TypeString},
+						Optional: true,
+					},
+					string(RollConfig): {
+						Type:     schema.TypeList,
+						Optional: true,
+						MaxItems: 1,
+						Elem: &schema.Resource{
+							Schema: map[string]*schema.Schema{
+								string(BatchSizePercentage): {
+									Type:     schema.TypeInt,
+									Required: true,
```

Comment:
please re-check, according to the document it is not required param, and in-case of null it will set to 20% default in the backend